pocs/utils/images/__init__.py

```diff
@@ -103,14 +104,22 @@ def _make_pretty_from_fits(fname, **kwargs):
         ax = plt.subplot(projection=wcs)
 
         ax.coords.grid(True, color='white', ls='solid')
-        ax.set_xlabel('RA')
-        ax.set_ylabel('Dec')
+
+        ra = ax.coords[0]
+        dec = ax.coords[1]
+
+        ra.set_axislabel('Right Ascension')
+        dec.set_axislabel('Declination')
+
+        ra.set_major_formatter('hh:mm:ss')
+        dec.set_major_formatter('dd:mm:ss')
     else:
         ax = plt.subplot()
 
         ax.grid(True, color='white', ls='solid')
-        ax.set_xlabel('Pixel x-axis')
-        ax.set_ylabel('Pixel y-axis')
+
+        ax.coords[0].set_axislabel('Pixel x-axis')
```

Comment:
As per comments above, I would just do `X [pixels]` and `Y [pixels]`.

Reply:
Or `X / pixels`, `Y / pixels` ;-)